src/server/boot/02-new-access-control.js

```diff
@@ -9,11 +10,11 @@ export default function(app) {
 
   // Endpoints for testing
 
-  app.get('/api/test/rbac-test-success', app.requirePermission('perform allowed test action'), (req, res) => {
+  app.get('/api/test/rbac-test-success', optionalBasicAuth(), app.requirePermission('perform allowed test action'), (req, res) => {
     res.send('You should see this!');
   });
 
-  app.get('/api/test/rbac-test-fail', app.requirePermission('perform disallowed test action'), (req, res) => {
+  app.get('/api/test/rbac-test-fail', optionalBasicAuth(), app.requirePermission('perform disallowed test action'), (req, res) => {
```

Comment:
optionalBasicAuthin voisi siirtää app.requirePermissionin sisään, jotta sitä ei tarvii toistaa joka paikassa

Reply:
Ainakin täällä oli joku tämmönen esimerkki: https://dsackerman.com/passportjs-using-multiple-strategies-on-the-same-endpoint/

Löysin joskus jonkun elegantimmankin version mut en löydä sitä enää :(